apps/association_result_annotation/dxapp.json

```diff
@@ -124,24 +148,67 @@
     {
       "name": "out_file",
       "label": "Output File",
-      "class": "file"
+      "patterns": ["*.tsv" ],
+      "class": "file",
+      "help": "The output file contains columns from the input file with additional columns requested for annotation."
     }
   ],
   "runSpec": {
     "execDepends": [
       {"name": "python-apsw"}
     ],
     "interpreter": "bash",
-    "file": "src/association_result_annotation.sh"
-  },
-  "regionalOptions": {
-    "aws:us-east-1": {
-      "resources": "project-BYpFk1Q0pB0xzQY8ZxgJFv1V"
+    "file": "src/association_result_annotation.sh",
+    "timeoutPolicy": {
+      "main": { 
+        "hours": 12  
+      },
+      "*": {
+        "hours": 8,
+        "minutes": 30
+      }
     }
+   },
+   "regionalOptions": {
+    "aws:us-east-1": {
+      "systemRequirements": {
+        "*": {
+          "instanceType": "mem1_ssd1_x4"
+        }   
+      },  
+      "assetDepends": [
+          {   
```

Comment:
Please update record IDs to:
aws:us-east-1: record-FJ3zf8j08zVZFzqYG88Pbk7z
azure:westus: record-FJ4058j9Vf9ZBBVB2Q7b3VQz